opensrp-chw-core/src/main/java/org/smartregister/chw/core/dao/AncDao.java

```diff
@@ -37,7 +37,7 @@ public static MemberObject getMember(String baseEntityID) {
                 "m.phone_number , m.other_phone_number , f.first_name family_name , f.primary_caregiver , f.family_head , " +
                 "fh.first_name family_head_first_name , fh.middle_name family_head_middle_name, fh.last_name family_head_last_name, " +
                 "fh.phone_number family_head_phone_number , ar.confirmed_visits , f.village_town , ar.last_interacted_with , " +
-                "ar.last_contact_visit , ar.visit_not_done , ar.last_menstrual_period  , al.date_created  , ar.* " +
+                "ar.last_contact_visit , ar.visit_not_done , ar.last_menstrual_period  , ar.gravida, al.date_created  , ar.* " +
```

Comment:
since we have **_ar.*_**  I think we don't need **ar.last_contact_visit , ar.visit_not_done , ar.last_menstrual_period  , ar.gravida**

Reply:
Cool. Makes sense. Let me test without the explicit column definitions